rust/core-lib/src/tabs.rs

```diff
@@ -741,6 +795,41 @@ impl CoreState {
         }
     }
 
+    #[cfg(feature = "notify")]
+    fn handle_open_file_tail_fs_event(&mut self, event: Event) {
+        use notify::event::*;
+
+        let path = match event.kind {
+            EventKind::Modify(ModifyKind::Any) => &event.paths[0],
```

Comment:
I think I need to handle other kinds of event here too. Im playing around with this.

Reply:
What other events do you have in mind?

Reply:
None actually. I forgot to update this comment.